### ISSUE TITLE
First Cycle: Dewey and Potato Leek Soup

### ISSUE THREAD

**Issue by norankhan** [processpark/chanorth2018-v.1#1]
John Dewey, Having an Experience, from Art as Experience (36, 37).

::
Experience occurs continuously, because the interaction of live creature and environing conditions is involved in the very process of living. Under conditions of resistance and conflict, aspects and elements of the self and the world that are implicated in this interaction qualify experience with emotions and ideas so that conscious intent emerges. Oftentimes, however, the experience had is inchoate. Things are experienced but not in such a way that they are composed into <em>an</em> experience. <strong>There is distraction and dispersion; what we observe and what we think, what we desire and what we get, are at odds with each other.</strong> We put our hands to the plow and turn back; we start and then we stop, not because the experience has reached the end for the sake of which it was initiated but because of extraneous interruptions or of inner lethargy.

::
<img src="http://www.joelkuennen.com/processpark/images/IMG_8128.JPG"/>
<img src="http://www.joelkuennen.com/processpark/images/IMG_8129.JPG"/>

Consideration/Contemplation: Add any thoughts and generate content in your own column under <a href="https://github.com/processpark/chanorth2018-v.1/projects/1">First Cycle Project</a>.

Potato Leek Soup - Ursula Endlicher

Sauté in olive oil while sipping some coffee
{
celery root
onions, white and red
carrots
}

Add:
{
Thyme from the foundation of the old summer kitchen, 2 Bay leaves, chopped parsley, three cloves garlic chopped, water
}
Simmer: for 1.5 hours

Sauté four leeks separately last to add after pureeing the soup.

**Comment by jjuliaa** [comment]
@ursula:
the 'timestep' in a computational system as an experience, latency between different experiences of time...

and..
some sorta okay examples of visualizing convolutional neural nets (regarding ML & seeing the Processes happen)
http://scs.ryerson.ca/~aharley/vis/conv/
https://playground.tensorflow.org/

there's another one, not finding now.

**Comment by santinaamato** [comment]
The search for a limitless experience within the realm of intimacy.

![fullsizerender 9](https://user-images.githubusercontent.com/38108648/38375190-37a91284-38bb-11e8-8fab-97b402063c1b.jpg)

**Comment by jjuliaa** [comment]
Poetry By Alex Cruse. 

Especially Page 2, 

**"A dollar is a machine
disguised as a picture"**

I think that part of the poem relates to this idea of economy/class and experience that we discussed. We are in an economy of Experience. Is the choice over experiences class based, or are there choices at all levels? It also makes me think of how technological mediation is used as escapism, or plays into the experience economy. 
The future of labor in an experience economy (once most jobs have been automated) is to make experiences available. The concept of choosing and "Making" your own experience, seems superficial or isn't accounting for the depth of biological systems, we aren't yet choosing our chemical composition. 
How to value, get complete full-fillment, from the experience of everyday being-ness, and is this practice a way to  circumvent the forces of "manufactured desire" (these are solved via some product, such as: desire for limitlessness (VR, internet, airbnb), desire not to be in a body that ages (VR, medicalization), desire to know everything and predict one's daily activities in advance(internet, google, IoT) that come as part of capitalism/late capitalism/experience-economies? Can you truly ever be the sole origin of your desires and experiences, we live in a world, you require oxygen? This idea comes from deep individualism and a discretization of and from the systems that surround, compose us and influence us. Also, Limits for me have been productive of imaginative thinking, and constraints often drive creative realization. 


Other thoughts: The experience of understanding. The mind's selectivity with regard to memory, ie) you only remember certain kinds of experiences...

<img width="549" alt="screen shot 2018-04-06 at 1 39 45 pm" src="https://user-images.githubusercontent.com/2681078/38435656-4eba5b78-39a0-11e8-933c-191af2ed20ba.png">
<img width="583" alt="screen shot 2018-04-06 at 1 39 15 pm" src="https://user-images.githubusercontent.com/2681078/38435657-4ece5e02-39a0-11e8-972c-d95679f4053a.png">
<img width="622" alt="screen shot 2018-04-06 at 1 38 56 pm" src="https://user-images.githubusercontent.com/2681078/38435658-4ee29c1e-39a0-11e8-9f7d-3a6b33264609.png">

**Comment by ursend** [comment]
2 loose thoughts:

1. How are experiences transcribed? As a a score, a story, a script… What comes first, the experience or the code? From one medium to the other (scripts, web-browsers,...) or from one person to the other (telling a personal story...). What comes first, the experience or its description? When writing code for a piece to be displayed to come "alive". When writing a recipe after/before preparing a dish? Is the description an experience? 


2. How did it become important to compress time in languages? Thinking of the french term “quatre-vingt” for eighty… or a “dozen” for 12… Because of the need to say thing easier and faster…? Is this what we are doing more and more, with acronyms of all sorts. 
--> In the mood for prolonged terminology… endless pictograms of  - an experience?

**Comment by norankhan** [comment]
@ursend, @jjuliaa, and @santinaamato: thank you! I am slowly #processing these fantastic contributions. I would like to pull elements out while talking tonight. See you soon.

**Comment by norankhan** [comment]
I felt the Dewey helped us start to sort through the many kinds of ways we talk about “experience”: aesthetic, interpersonal, interior, cultural. We discussed what makes an experience be high quality or low, how we know we are even having one, how material limits allow for experience or snuff them out in their cribs, and what a limitless life or limitless experience might even feel like. We distinguished, it seems, as a group, between Dewey’s process of experience as meandering and an experience as defined by its closure, its cessation. And we also talked about how to make meaningful experiences for ourselves, not driven necessarily by a manic feeling or intuition, but instead by their usefulness, by their repetition. Creating deeper grooves in the brain through that repetition. Allowing the experience of claymaking or marking the canvas or page to gain meaning through the repetition. 

_Transcription_: For me, I think of some of the big “important” experiences of my life – seeing Lang Lang play Rachmaninoff when inconsolable, reading Paradise Lost and making a stupid plan to tattoo the passage on the muse on my arm, visiting Hiroshima when 13, telling a friend I loved them, helping a friend who is suicidal – and I hold them as profoundly personal, almost unspeakable. But I have to convey them to others, it seems, for them to gain meaning. The transcription gives the experience meaning; perhaps the experience is equally in the act of transcribing it, interpreting it, and explaining it to others. This is what artists are also doing constantly – transcribing their inner experiences, their dream states, their inner thinking process – into representation. 

@ursend asks, right on time, “How are experiences transcribed? … What comes first, the experience or its description?” The description is itself an experience – and we can debate this – I’d argue the experience is made through description. Maybe I argue that we need language because I need a job. I once I got into it with a friend who said that “language ruins experience.” That there’s no way language can capture the core, the essence! 

Visual artists feel this struggle acutely when demanded what their work means – think of Dick in I Love Dick turning away from a writer or shrugging when asked earnest questions from fans. There’s a kind of violence in the act of description, a series of choices about what happened, about what was significant, but without that narration the experience risks slipping away into the ether.

_Unknowing and Mess_: Even in transcription, there is a profound messiness. My private, non- or pre-verbal experiences can be an amalgam of religious ecstasy and base sensual violence and fantasies of divine knowledge, but if I were to say that, I sound high. I cut out the unknowing and mess through clean language – and the asymptote, the limit, towards a clear, perfect representation, is worth one’s whole life of work. Alternately, if I want to convey that mess, I can be messier, more experimental. Not following a linear path, not going from A to B, but instead cycling, digressing, meandering …

I love the loss, and the messiness. I long to restore the loss, and recreate the mess. I think of translating experiences to capture a loss just as people obsessively work through their dreams the morning after, chasing a receding horizon and fragmentary images. There’s also beauty in the compression into language, as Ursula writes. There’s a knowledge that the core of the experience will never be expressed, but we work around it in language, and in relating the experience to others. We have evolved to need symbols and acronyms and shorthand, pictograms and representation. We need to try to freeze the experience, come up with new pictograms and symbols, new stories, as the original experience recedes further in our memory. What remains is the representation.

_Relationality_: Relationality! We can have the exact same experience and one of us think it life-changing and significant and the other think it total shit. Or be unable to process it in the same way because of different life experiences, backgrounds, tools, languages. If I am across from you around an object, a piece of art, say, we share what we see, what we feel looking at it. We have selective memories about experiences, as Jules noted. We learn through the relational attempt to describe the experience together. It reveals how we also might have an experience based on any complex number of emotional factors, states of mind, chemical depressions or elevations; it might mean a great deal today, and tomorrow, we’ll ask, what’s the point of any of it?

_Limitlessness_: “The search for a limitless experience within the realm of intimacy,” from @santinaamato . I love this – and want to return to it many times – the idea of a relationship that expands you, an intimacy through which all things feel possible. This is how I think of love – through your partner or friend, or whoever you might feel or have felt intimate with – there is a sense of the endlessly possible. The possibility of a field, of many fields. I know I’m in love when the banal seems exquisite and every door leads to ten more and ten more and ten more. There feels like no end to the conversation. I’m adding this Mary Reufle quote from _Madness, Rack, and Honey_:

![limitlessness](https://user-images.githubusercontent.com/15232870/38461313-ad422e8c-3a9a-11e8-9806-1bdfadca700f.png)

_Privilege_: @jjuliaa wrote about the economy of experience, and I think a lot about how class and financial opportunity limit experiences, access – to culture, to thinking, to education – which means limits on experience. From limits of experience, comes limits on perspective. Even the ability to get value from being in the everyday, treating it as a practice – this comes from education. We talk about curating experiences, making our experiences – but there are sociological factors, environment, any number of issues out of our power that define experiences and their quality, as much a product of systems as separate from them.

Late capitalism’s constraints have to be productive for creative people, who have no choice but to work within its bounds. Even going out into the woods is made possible by late capitalism. I think of Mark Corrigan in Peep Show, after a long night of ravers drugging in the apartment: https://www.youtube.com/watch?v=5D1gJ_GygAI, on the miracle of consumer capitalism. 

The very option of limitless experiences are also made possible by systems that privilege some while crushing others. Prediction, mapping, quantifying intensity of experiences: intensity is the metric. I like Jules’ points – that this search for limitlessness can be solipsistic, product of a culture that allows and encourages that pursuit.

_Repetition_: The process of mapping the brain allows us to see the grooves in the brain being dug. Mapping and visualizing how the mind can figure out new pathways, new modes of navigating. We can create experience intentionally by training our brains through repetition, building new paths. Since the brain loses plasticity over time – we can seize on this ability to cut new pathways. People do crosswords daily "to keep the mind active.” To think that we are about 10 or 15 repetitions away from having experiences that make us feel good, and not down. In the thicker lines, a practice.

I like the idea of us building a neural network together with our own trained set of data. What data would we pick? What problem would we ask it to solve? Could you train a brain to be more open to intimacy, to enjoy multiplicity, to be oriented towards limitless experiences?

![paths and reward](https://user-images.githubusercontent.com/15232870/38461318-c04a20f2-3a9a-11e8-9e16-41216428d0fb.png)

**Comment by processpark** [comment]
One of the most interesting phrases in Dewey's description of experience is the first phrase; "Experience occurs continuously...". As we begin to see experience as dependent on a past condition, an experience _has happened_, the opening phrase suggests the kind of interleaved layering characteristic of life. Experiences leaved one on top of another, haphazardly overlaying into a growing palimpsest of meanings, interrelated experiences orienting the self around the shaded portions of Ven diagrams. From this form of triangulated meaning, "conscious intent" emerges.

What is an experience that doesn't become an experience? What is forgetting? What isn't an experience?
